starter_templates/cpp/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+
+int main() {
+  // You can use print statements as follows for debugging, they'll be visible when running tests.
+  std::cout << "Logs from your program will appear here!\n";
+
+  // Uncomment this block to pass the first stage
+  // std::cout << "$ ";
+  // std::cout.flush();
```

Comment:
Can we flush `cerr` too?
This is what we have been doing in all other courses. 

```// Flush after every std::cout / std::cerr
std::cout << std::unitbuf;
std::cerr << std::unitbuf;
```

I feel this block can be at the top, without commenting it out.